docs/how-to-build-with-ddn/with-stripe.mdx

```diff
@@ -0,0 +1,243 @@
+
+### Step 4. Introspect your Stripe account
+
+```sh title="Next, use the CLI to introspect your Stripe account:"
+ddn connector introspect my_stripe
+```
+
+After running this, you should see a representation of the Stripe API's schema in the `app/metadata/my_stripe.hml` file;
+you can view this using `cat` or open the file in your editor.
+
+Under the hood, the Stripe connector utilizes the [`ndc-http` connector](https://github.com/hasura/ndc-http) via
+[Stripe's OpenAPI spec](https://github.com/stripe/openapi). This ensures you can incorporate any resources available via
+Stripe's REST API into your supergraph.
+
+```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
+ddn connector show-resources my_stripe
+```
+
+### Step 5. Add your command
+
+```sh title="Let's start by tracking the GetProducts endpoint from Stripe's API:"
+ddn command add my_stripe "GetProducts"
+```
+
+Open the `app/metadata` directory and you'll find a newly-generated file: `GetProducts.hml`. The DDN CLI will use this
+Hasura Metadata Language file to represent the `GetProducts` endpoint from Stripe in your API as a
+[command](/reference/metadata-reference/commands.mdx).
+
+### Step 6. Create a new build
+
+```sh title="To create a local build, run:"
+ddn supergraph build local
+```
+
+The build is stored as a set of JSON files in `engine/build`.
+
+### Step 7. Start your local services
+
+```sh title="Start your local Hasura DDN Engine and Stripe connector:"
+ddn run docker-start
+```
+
+Your terminal will be taken over by logs for the different services.
+
+### Step 8. Run your first query
+
+```sh title="In a new terminal tab, open your local console:"
+ddn console --local
+```
+
+```graphql title="In the GraphiQL explorer of the console, write this query:"
+query GET_ALL_PRODUCTS {
+  getProducts {
+    data {
+      id
+      name
+      description
+    }
+  }
+}
+```
+
+```json title="You'll get a response listing all of your products. For example:"
+{
+  "data": {
+    "getProducts": {
+      "data": [
+        {
+          "id": "3bef8a40-3c33-11ee-bb29-070df467ec94",
+          "name": "Tee",
+          "description": "A great t-shirt!"
+        },
+        {
+          "id": "e0a70b16-65b6-11ed-8788-8fa2504d64a3",
+          "name": "Sticker Sheet",
+          "description": "Who doesn't love stickers??"
+        },
+        {
+          "id": "a44eda7c-65b6-11ed-997b-53b5bdb7117e",
+          "name": "Hasuras in The Cloud Tee",
+          "description": "The cloud for your cloud."
+        },
+        {
+          "id": "8aa93f86-65b6-11ed-901c-f320d4e17bb2",
+          "name": "Dark Furry Logo Tee",
+          "description": "Furry for furries."
+        }
+      ]
+    }
+  }
+}
+```
+
+### Step 9. Add another command
+
+```sh title="Add the GetCheckoutSessions command:"
+ddn command add my_stripe "GetCheckoutSessions"
```

Comment:
Would it be more helpful here to show a mutation? PostProducts or PostCoupons maybe simpler.

Reply:
We definitely could, but I held off since Stripe's mutations generally require a number of arguments/fields. What I will do is make mention that you can directly insert, update, and delete data using the connector 👍